docs/ops/logging/authentication.rst

```diff
@@ -0,0 +1,58 @@
+############################
+Elasticsearch authentication
+############################
+
+Elasticsearch has a blizzard of authentication options, and Open Distro for Elasticsearch replaces those with yet different ones.
+They're also in general poorly documented.
+This is an attempt to explain the basics of the authentication system and the choices we made when configuring it.
+
+.. rubric:: Configuration
```

Comment:
In this page you can switch over to regular sectioning rather than the `rubric` directive.